Description: Upstream changes introduced in version 0.01
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 classicmenu-indicator (0.01) natty; urgency=low
 .
   * Initial release.
 .
 The person named in the Author field signed this changelog entry.
Author: Florian Diesch <debian@florian-diesch.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ classicmenu-indicator-0.01/Makefile
@@ -0,0 +1,29 @@
+NAME=classicmenu-indicator
+VERSION=0.01
+DEBVERSION=0.01
+PPA=diesch/testing
+
+.PHONY: clean deb sdist ppa deb
+
+
+clean:
+	rm -rf *.pyc build dist  ../${NAME}_${DEBVERSION}* classicmenu_indicator.egg-info
+
+
+sdist:
+	python setup.py sdist
+
+egg:
+	python setup.py bdist_egg
+
+sdeb: sdist
+	cp dist/${NAME}-${VERSION}.tar.gz ../${NAME}_${VERSION}.orig.tar.gz
+	rm -r dist
+	debuild -S -sa  -v${DEBVERSION} -kB57F5641
+
+deb: sdeb
+	debuild -b -sa -v${DEBVERSION} -kB57F5641
+
+ppa: sdeb
+	dput ppa:${PPA} ../${NAME}_${DEBVERSION}_source.changes
+
--- /dev/null
+++ classicmenu-indicator-0.01/entry.dir
@@ -0,0 +1 @@
+['__class__', '__delattr__', '__doc__', '__format__', '__getattribute__', '__hash__', '__init__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', 'comment', 'desktop_file_id', 'desktop_file_path', 'exec_info', 'get_comment', 'get_desktop_file_id', 'get_desktop_file_path', 'get_display_name', 'get_exec', 'get_generic_name', 'get_icon', 'get_is_excluded', 'get_is_nodisplay', 'get_launch_in_terminal', 'get_name', 'get_parent', 'get_type', 'icon', 'is_excluded', 'is_nodisplay', 'launch_in_terminal', 'name', 'parent', 'type']
--- /dev/null
+++ classicmenu-indicator-0.01/desktop/autostart/classicmenu-indicator.desktop
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=ClassicMenu Indicator
+Comment=Indicator applet to show the Gnome Classic main menu
+GenericName=ClassicMenu Indicator
+Categories=GNOME;Utility;
+Exec=classicmenu-indicator
+Icon=gnome-main-menu
+Type=Application
--- /dev/null
+++ classicmenu-indicator-0.01/desktop/application/classicmenu-indicator.desktop
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=ClassicMenu Indicator
+Comment=Indicator applet to show the Gnome Classic main menu
+Comment[de]=Ein Indikator-Applet, das das Hauptmenu von Gnome Classic anzeigt
+GenericName=ClassicMenu Indicator
+Categories=GNOME;Utility;
+Exec=classicmenu-indicator
+Icon=gnome-main-menu
+Terminal=false
+Type=Application
